README.md

```diff
@@ -54,13 +54,13 @@ available at <https://lore.kernel.org/git/>,
 <http://marc.info/?l=git> and other archival sites.
 The core git mailing list is plain text (no HTML!).
 
-Issues which are security relevant should be disclosed privately to
+Issues which are security relevant and should be disclosed privately to
 the Git Security mailing list <git-security@googlegroups.com>.
 
 The maintainer frequently sends the "What's cooking" reports that
 list the current status of various development topics to the mailing
 list.  The discussion following them give a good reference for
-project status, development direction and remaining tasks.
+project status, development direction and the remaining tasks.
```

Comment:
The original version sounds more open-ended, which reflects reality a bit better, I think.